src/Components/DrinkDetails/DrinkDetails.tsx

```diff
@@ -25,10 +22,12 @@ class DrinkDetails extends Component<any, any> {
                 drink: data
             })
         })
+            .catch((error) => {
+                throw new Error('Error')
```

Comment:
A catch... catches (couldn't think of another word there) an object of the Error class and then does something as a result of that error, usually providing some feedback to the user to let them know what's gone wrong.  

Basically, if we throw an Error here again we're just tossing it down the line again, which would need to be resolved by a second catch I believe. Instead, we probably want to mimic what we've done in other files and have a state for error, which populates with some message (I believe we're just using "Server error." elsewhere and can do so here as well) and triggers a conditional redirect to the error page.

I think the DrinkGrid component might have an example of this? Sorry for a very long comment, but hopefully that's somewhat helpful.